mvp.feature

```diff
@@ -865,11 +866,12 @@ Feature: MVP features for NVA
   # Actions from Page : My Publications (Delete)
   @356
   Scenario: Creator deletes an item in My Publications list
-    Given that the user is logged in as Creator
-    And is on the My Publications page
+    Given the user is logged in as Creator
+    And has opened the page My Publications
```

Comment:
kan bytte ut denne Given'en med den forrige, og slette disse to linjene.
`Given Creator opens My Publications`